src/BloomExe/Edit/EditingView.cs

```diff
@@ -1039,5 +1042,15 @@ public bool ChangingPages
 				_pageListView.Cursor = Cursor;
 			} 
 		}
+
+		public void ShowAddPageDialog()
+		{
+//			if(_view == null || _inProcessOfDeleting || _addPageDialogShowing)
+//				return;
+			//_addPageDialogShowing = true;
+			var jsonTemplates = _model.GetTemplateBookInfo;
+			//if the dialog is already showing, it is up to this method we're calling to detect that and ignore our request
```

Comment:
Is it worth the extra precision to say, "if ANY dialog is..."? e.g. if we changed the format dialog to use this library, you wouldn't be able to add a page while doing formatting, and the more precise comment might help someone realize why.